src/config/services/index.md

```diff
@@ -3,19 +3,33 @@
 Void uses the [runit(8)](https://man.voidlinux.org/runit.8) supervision suite to
 run system services and daemons.
 
+## Enable
+
 Services are enabled by simply linking them into the `/var/service` service
 directory.
 
 ```
 # ln -s /etc/sv/<service name> /var/service/
 ```
 
+## Disable
+
 To disable them again you just remove the link.
 
 ```
 # rm /var/service/<service name>
 ```
 
+## Disable Autorun
```

Comment:
Maybe this should be reworded a bit, but I think it's a useful addition

Reply:
I think this is a good addition, but maybe we can leave the disabling heading and remove this heading. And a little rewording would be useful. "To disable a service from starting at boot, while still being managed by runit, create a file named `down` in it's service directory." Or something.